bom/pom.xml

```diff
@@ -536,7 +534,6 @@
                 <groupId>io.quarkus.gizmo</groupId>
                 <artifactId>gizmo</artifactId>
                 <type>test-jar</type>
-                <scope>test</scope>
                 <version>${gizmo.version}</version>
```

Comment:
We don't use this dependency anywhere? Makes sense to have it anyway but we need to make sure we didn't miss a scope.

Reply:
It's used in `core/deployment`

Reply:
The code was already using the `test` scope.